### PR TITLE
Initial Workshop Changes

### PR DESCRIPTION
**Pull request by GannonGuess**
First batch of changes to the PDAL Workshop that I thought were worthy of implementation.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -173,3 +173,4 @@ test/data/*.json
 test/data/pipeline/*.json
 test/data/filters/*.json
 test/data/plang/*.json
+doc/workshop/exercises/workshop-updates.md
```
Comment (j9ac9k) [inline .gitignore:176]: Why is this file added to `.gitignore`  I would think this file we want to be in the repo?

File: doc/workshop/exercises/analysis/denoising/denoising.rst [hunk 2]
```diff
@@ -132,7 +132,7 @@ opened the file using the `Fugro Viewer`_
 .. image:: ../../../images/denoise-fugro.png
     :target: ../../../../_images/denoise-fugro.png
 
-.. _`Fugro Viewer`: http://www.fugroviewer.com/
+.. _`Fugro Viewer`: https://hub.fugro.com/fugroviewer_socialpg/?utm_source=DSMN8&utm_medium=LinkedIn/
```
Comment (j9ac9k) [inline doc/workshop/exercises/analysis/denoising/denoising.rst:135]: we can probably remove everything in the URL from the `?` on.

File: doc/workshop/exercises/analysis/dtm/dtm.rst [hunk 3]
```diff
@@ -103,9 +101,6 @@ see what was produced. Let us use :ref:`qgis` to visualize the output.
     .. image:: ../../../images/dtm-qgis-classify.png
         :target: ../../../../_images/dtm-qgis-classify.png
 
-    .. image:: ../../../images/dtm-qgis-colorize-dtm.png
-        :target: ../../../../_images/dtm-qgis-colorize-dtm.png
```
Comment (j9ac9k) [inline doc/workshop/exercises/analysis/dtm/dtm.rst:107]: Does this image need to be removed from the repo as well?

**Comment by hobu** [comment]
picked back to `2.5-maintenanced` at d674e3a4862b4a2caeef64e91f81ae5f5d3ee55d
